tests/Oryx.BuildImage.Tests/BenvTest.cs

```diff
@@ -138,6 +138,32 @@ public void UserInstalledExecutable_IsChosenOverOryxExecutable_InVsoBuildImage(s
                 result.GetDebugInfo());
         }
 
+        [Theory]
+        [InlineData("python", "/usr/bin/")]
+        [InlineData("python3", "/usr/bin/")]
+        [InlineData("pip", "/usr/local/bin/")]
+        [InlineData("pip3", "/usr/local/bin/")]
```

Comment:
add specific suffix at the path, like `/usr/bin/python`,  `/usr/local/bin/pip3`

Reply:
Sure, sounds good